src/Controller/HearingController.php

```diff
@@ -543,6 +616,22 @@ public function hearingPostDelete(CaseEntity $case, HearingPost $hearingPost, Re
                     $this->entityManager->remove($hearingRecipient);
                     $this->entityManager->remove($document);
                 }
+
+                if ($briefing = $hearingPost->getBriefing()) {
+                    $briefingRecipients = $briefing->getHearingBriefingRecipients();
+                    foreach ($briefingRecipients as $briefingRecipient) {
+                        $document = $briefingRecipient->getDocument();
+
+                        $this->removeDocumentFromCase($case, $document);
+
+                        $this->entityManager->remove($document);
+                        $this->entityManager->remove($briefingRecipient);
+                    }
```

Comment:
This code largely duplicates the code in lines 421–433. Does it make sense to create a helper method for this.

Reply:
Fixed.